airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/test/steps/python_connectors.py

```diff
@@ -168,7 +168,7 @@ async def install_testing_environment(
         )
         if self.common_test_dependencies:
             container_with_test_deps = container_with_test_deps.with_user("root").with_exec(
-                ["pip", "install", f'{" ".join(self.common_test_dependencies)}']
+                ["pip", "install"] + self.common_test_dependencies
```

Comment:
This was running the command as `pip install '<dep1> <dep2>'` which was causing errors the current solution would return `pip install <dep1> <dep2>` instead

Reply:
To my eyes, this looks like the right fix. 👍